stderr of test 'sample0` in directory 'sql/test/mapi` itself:


# 22:20:44 >  
# 22:20:44 >  Mtimeout -timeout 180 Mserver "--config=/var/tmp/_PREFIX_MONET_GNU_32_d__/etc/monet.conf" --debug=10 --set "monet_mod_path=/var/tmp/_PREFIX_MONET_GNU_32_d__/lib/MonetDB" --set "gdk_dbfarm=/var/tmp/_PREFIX_MONET_GNU_32_d__/var/MonetDB/dbfarm" --set "sql_logdir=/var/tmp/_PREFIX_MONET_GNU_32_d__/var/MonetDB/log" --set mapi_port=49241 --set sql_port=58277 --set monet_prompt= --trace "--dbname=mTests_src_mapi_examples_C" "/var/tmp/_PREFIX_MONET_GNU_32_d__/lib/MonetDB/mapi.mil" ; echo ; echo Over..
# 22:20:44 >  


# 22:20:45 >  
# 22:20:45 >  Mtimeout -timeout 60 ./sample0.MAPI sample0 
# 22:20:45 >  


# 22:20:45 >  
# 22:20:45 >  Mtimeout -timeout 60 Mshutdown.py "--config=/var/tmp/_PREFIX_MONET_GNU_32_d__/etc/monet.conf" --host=draco --port=49241
# 22:20:45 >  


# 22:20:46 >  
# 22:20:46 >  Done.
# 22:20:46 >  

